src/manifest.json

```diff
@@ -35,10 +47,10 @@
     }
   },
 
-  "__chrome__minimum_chrome_version": "49",
+  "__chrome__minimum_chrome_version": "88",
   "__opera__minimum_opera_version": "36",
```

Comment:
This version should be updated as well

Reply:
https://en.wikipedia.org/wiki/History_of_the_Opera_web_browser#Opera_2020
v74 is based on chrome v88.
Updating to 74.